bouncer/blacklist/tests.py

```diff
@@ -137,3 +138,30 @@ def test_is_email_host_blacklisted_with_not_blacklisted_host(self):
         """
 
         self.assertFalse(is_email_host_blacklisted(self.not_blacklisted_host))
+
+
+class IPRequestViewTests(TestCase):
```

Comment:
You can create a base class, which contains `setUpClass`, to make sure you don't repeat yourself in every test case 😇